2019-11-16 01:34:34.471 - [Info] MITM Version: 1.26
2019-11-16 01:34:34.478 - [Info] Auto Access Enabled: true
2019-11-16 01:34:34.478 - [Debug] [Init] Auto Access Barrier: true
2019-11-16 01:34:34.501 - [Info] SSH man-in-the-middle server for 172.20.0.2 listening on 0.0.0.0:10101
2019-11-16 01:58:00.219 - [Debug] [Connection] Attacker connected: 211.138.181.202 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-16 01:58:03.475 - [Debug] [Auth] Attacker 211.138.181.202 trying to authenticate with "password"
2019-11-16 01:58:03.477 - [Debug] [Auto Access] Attacker: 211.138.181.202, Threshold: 1, Attempts: 1
2019-11-16 01:58:03.478 - [Debug] [Auto Access] Compromising the honeypot
2019-11-16 01:58:03.478 - [Debug] [Auto Access] Adding the following credentials: "root:jiangwei521"
2019-11-16 01:58:03.749 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-16 01:58:03.749 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.2
2019-11-16 01:58:03.880 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-16 01:58:05.465 - [Debug] [Connection] Attacker connected: 211.138.181.202 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-16 01:58:07.780 - [Debug] [Auth] Attacker 211.138.181.202 trying to authenticate with "password"
2019-11-16 01:58:07.782 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-16 01:58:07.784 - [Debug] [Auth] Attacker: 211.138.181.202 has so far made 1 attempts. Remaining: 5 attempts
2019-11-16 01:58:08.883 - [Debug] [Connection] Attacker closed the connection
2019-11-16 01:58:09.399 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-16 02:02:45.218 - [Debug] [Connection] Client error on ssh server
2019-11-16 02:02:45.219 - [Debug] [Connection] Attacker closed connection
2019-11-16 02:02:45.225 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-16 02:02:45.226 - [Debug] [LXC] Container's OpenSSH server closed connection
